src/bin/edit/localization.rs

```diff
@@ -675,7 +675,7 @@ const S_LANG_LUT: [[&str; LangId::Count as usize]; LocId::Count as usize] = [
     // LargeClipboardWarningLine3
     [
         /* en      */ "Do you want to send it anyway?",
-        /* de      */ "Möchten Sie es trotzdem senden?",
+        /* de      */ "Möchten Sie die Daten trotzdem senden?",
```

Comment:
Perhaps "den Text" instead to refer to "Text" in `LargeClipboardWarningLine1`?

Reply:
In the context of seeing all 3 lines of text next to each other, I think "es" is the best, because at least to me it mistakenly refers to the text that will be sent.

But if you feel like that it's too vague, I'm happy to go with "die Daten" for now.